src/libraries/System.Runtime.InteropServices/System.Runtime.InteropServices.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32211.407
+MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
I assume VS did this? Do we try to keep the .sln files slngen edits only?

Reply:
Generally we try to keep the sln files such that they don't generate diffs when you open them in VS, but that usually depends on slngen generating them perfectly and sometimes it doesn't do it quite right.